tools/importPlanGoals.js

```diff
@@ -0,0 +1,185 @@
+                      }
+                    });
+                  }
+                  // Add topic to junction with goal
+                  currentGoalId = cleanGoals.length + currentGoals.length - 1; // -1 because there is a dummy element at index 0
+                  cleanTopicGoals.push({ topicId, goalId: currentGoalId });
+                }
+              });
+            } else // it's either "timeframe" or "status"
+            // both "timeframe" and "status" column names will be reused as goal's object keys
+            if (currentGoalName !== '') {
+              currentGoal[column] = el[key].trim();
+              const goalNum = goalColumn[1].slice(0, 1); // represents a goal number from 1 to 5
+              currentGoals[goalNum] = { ...currentGoals[goalNum], ...currentGoal };
+            }
+          }
+        }
+      });
+
+      // after each row
+      let goalId;
+      let grantId;
+
+      currentGoals.forEach((goal) => {
+        cleanGoals.push(goal);
+        goalId = cleanGoals.length;
+        currentGranteeId = getItemId(cleanGrantees, currentGrantee.trim());
+        currentGrants.forEach((grant) => {
+          const fullGrant = { number: grant.trim(), granteeId: currentGranteeId };
+          if (!cleanGrants.some((e) => e.granteeId === fullGrant.granteeId
+                            && e.number === fullGrant.number)) {
+            cleanGrants.push(fullGrant);
+          }
+          grantId = cleanGrants.findIndex((g) => g.number === fullGrant.number) + 1;
+          const plan = { granteeId: currentGranteeId, grantId, goalId };
+          if (!cleanGrantGoals.some((e) => e.granteeId === currentGranteeId
+                            && e.grantId === grantId
+                            && e.goalId === goalId)) {
+            cleanGrantGoals.push(plan);
+          }
+        });
+      });
+    });
+
+    // The data has been prepared. Insert it into the database
+    await Role.bulkCreate(cleanRoles.map((role, index) => (
+      { id: index + 1, name: role })),
+    {
+      updateOnDuplicate: ['name', 'updatedAt'],
+    });
+    await Topic.bulkCreate(cleanTopics.map((topic, index) => (
+      { id: index + 1, name: topic })),
+    {
+      updateOnDuplicate: ['name', 'updatedAt'],
+    });
+    await RoleTopic.bulkCreate(cleanRoleTopics,
+      {
+        ignoreDuplicates: true,
+      });
+    await Goal.bulkCreate(cleanGoals.map((goal, index) => (
+      { id: index + 1, ...goal, isFromSmartsheetTtaPlan: true })),
+    {
+      updateOnDuplicate: ['name', 'updatedAt'],
+    });
+    await TopicGoal.bulkCreate(cleanTopicGoals, {
+      ignoreDuplicates: true,
+    });
+    await Grantee.bulkCreate(cleanGrantees.map((grantee, index) => (
+      { id: index + 1, name: grantee })),
+    {
+      updateOnDuplicate: ['name', 'updatedAt'],
+    });
+    await Grant.bulkCreate(cleanGrants.map((grant, index) => (
+      { id: index + 1, number: grant.number, granteeId: grant.granteeId })),
```

Comment:
If we're going to run this once for each region, it might be useful to be able to pass region as a command line argument to the import script (just like the `-f` arg) and then create grants with the proper region set.